### DISCUSSION TITLE
Erreur au niveau d'un mot

### DISCUSSION THREAD

**Discussion by Gamecraft400** [L-Sherry/French-CC#14] [General]
![image](https://user-images.githubusercontent.com/102668085/203837524-f88d29e9-2c09-4f3f-9647-89390d2097c1.png)
Ici à cette scene du jeu, ça doit être Scénario et non Scénarii

**Comment by L-Sherry** [comment]
En fait, le mot doit être au pluriel. Et le pluriel de "scénario" est un troll récurrent : pour certains c'est "scénarios" pour d'autre, c'est "scénarii" ou "scenarii".

https://fr.wiktionary.org/wiki/sc%C3%A9nario#Notes
https://fr.wiktionary.org/wiki/Discussion:scénario
J'ai l'impression que les "scénarii"/"scenarii" sont plus populaires chez les artistes... comme Gautham.
------- Original Message -------

**Reply by Gamecraft400** [reply]
OK, bah désolé pour le poste encore merci pour cette traduction au petit oignon au passage
